### PR TITLE
Bump version to 0.2.30

### PR DESCRIPTION
**Pull request by sid77**
This PR bumps gem version up to 0.2.30 and adds instructions about how to enable CSP support.

### CODE REVIEW

File: README.mdown [hunk 1]
```diff
@@ -194,6 +194,58 @@ You can also override `IntercomRails::Config` options such as your `api_secret`,
   }) %>
 <% end %>
 ```
+### Content Security Policy Level 2 (CSP) support
+As of version 0.2.30 this gem supports CSP, allowing to whitelist the include code using both nonces and SHA-256 hashes.
```
Comment (RuairiK) [inline README.mdown]: Small grammar point: it should read "allowing you to whitelist..."

File: Gemfile.lock [hunk 2]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    intercom-rails (0.2.29)
+    intercom-rails (0.2.30)
```
Comment (RuairiK) [inline Gemfile.lock]: I don't understand why we're editing the `Gemfile.lock` directly, this seems like bad practice. But I see that you're not the first one to do this :(.

**Comment by RuairiK** [comment]
One small correction, and a note, but I think you're good to :ship:

File: README.mdown (hunk 1 above)
Comment (sid77) [inline README.mdown]: fixed

File: Gemfile.lock (hunk 2 above)
Comment (sid77) [inline Gemfile.lock]: I was wondering about it as well! After some digging it seems it's considered a good practice, instead: https://github.com/github/gitignore/blob/master/Rails.gitignore#L23-L24

**Pull request merged**